tests/compile/callingconvention.srctree

```diff
@@ -35,10 +35,19 @@ p2 = f2
 p3 = f3
 p4 = f4
 
+
+######## callingconvention.h ########
+
+#define DLL_EXPORT
+#include "external_callingconvention.h"
+
+
 ######## external_callingconvention.h ########
 
 #ifndef DL_IMPORT
   #define DL_IMPORT(t) t
+#elif defined(DLL_EXPORT)
+  #define DL_IMPORT(t) DL_EXPORT(t)
```

Comment:
You might be misunderstanding the purpose of this but I'm not sure. Here `DLL_EXPORT` is a flag indicating that the header is being used to share declarations between translations units that will be statically linked (in which case all objects should be declared `DL_EXPORT`), as opposed to providing a declaration for an object that will be loaded by dynamic linking to a DLL (in which case it should be `DL_IMPORT`).  I.e. the whole discussion in #360.